en/api/configuration-generate.md

```diff
@@ -147,16 +139,14 @@ const axios = require('axios')
 
 module.exports = {
   generate: {
-    routes: function (callback) {
-      axios.get('https://my-api/users')
-      .then((res) => {
-        var routes = res.data.map((user) => {
-          return '/users/' + user.id
+    routes: callback =>
```

Comment:
Actually @manniL I now see that this may be confusing, because it does call the callback, but also returns a `Promise` which resolves after the callback returns.
Would putting `void axios.get` be too arcane?
Should I just put the body of the `routes` function into a `{}` block so that it returns `undefined`?

Reply:
Not sure if we want to keep a *callback* example for the routes function at all.

Reply:
That's fine by me. :weary: :ok_hand: 
So I nuke it?
From all languages?

Reply:
I'd say so